plugins/generator-1.19.4/forge-1.19.4/mappings/soundcategories.yaml

```diff
@@ -0,0 +1,33 @@
+_default:
+  - NEUTRAL
+  - neutral
+ambient:

```

Comment:
In vanilla code, the first element is MASTER, and AMBINED is 2nd last so this list needs some attention too